src/Components/Accounts/AccountList.js

```diff
@@ -70,9 +71,11 @@ class AccountList extends Component {
             <span
               className="ShowKeys"
               onClick={() => {
-                this.showKeys(
+                self.showKeys(
```

Comment:
Shouldn't `self` not be necessary here since there are two `=>` functions?

Reply:
It isn't - made this change while debugging something else to rule out the "wrong `this`" problem, because `this` was showing up as `undefined` at this line in the debugger. Ultimately that change was dumb because the error I was troubleshooting was occurring inside `showKeys`, proving that `this` was just fine. :-(